### ISSUE TITLE
Consolidate to single app

### ISSUE THREAD

**Issue by Datamance** [lookit/lookit-api#565]
**TL;DR**: Consolidate the various app directories - accounts, api, exp, studies, web, project - into a single app.

**Narrative**
As a developer, I want the organization of my app code to make sense. Furthermore, I don't want a complicated dependency graph for migrations.

The choice to split lookit-api into several applications was ill-advised. Given the high cohesion/interconnectedness of the data model, it has created more problems than solved by imposing awkward conceptual boundaries that have become a nuisance when adding new features ("does this go in `accounts` because it deals with accounts? Or does it go in `web` because it's part of the "web" interface? What does "web" even mean - isn't the entire application, a web application?).

We should consolidate to a single app, and in the process blow away the previous migrations and create a new initial migration. This could even coincide with our new semver'd v1 (#558), to formalize the notion that this is the real v1 data model.

**Acceptance Criteria**
A precise description of how/when this issue will be considered closed, e.g.:

- [ ] One app to rule them all
- [ ] All tests pass

**Implementation Notes**
This will also speed up tests (see #564 ), since we won't be forcing our test harness to dutifully recapitulate the entire evolution of Lookit's data model every time we run `python manage.py test`.
